  frame: /map
  #for segway
  image_topic_name: /segbot_cam/image_raw/compressed
  
  #for hexarotor
  #image_topic_name: /hexa_cam/image_raw/compressed
  
  sensor_field_of_view: 0.523
  senor_distance: 3.0
  sensor_angular_res: 0.05
